api-contracts/v1/workflows.proto

```diff
@@ -67,6 +68,15 @@ enum RateLimitDuration {
     YEAR = 6;
 }
 
+enum RunStatus {
+    QUEUED = 0;
+    RUNNING = 1;
+    COMPLETED = 2;
+    FAILED = 3;
+    CANCELLED = 4;
+}
```

Comment:
this is a bit icky since we already have this type on the OpenAPI side, but I wasn't sure if there's a better way